controllers/file.js

```diff
@@ -9,6 +9,11 @@ const fs = require('fs');
 const onFinished = require('on-finished');
 const config = require('config');
 const path = require('path');
+const {URL} = require('url');
+
+const crypto = require('crypto');
+const algorithm = 'aes-256-ctr';
+const password = config.get('aws.password');
 
```

Comment:
You _might_ want to consider throwing an error if the password is `changeme`. Bullet proof socks 'n' all that.

Reply:
will do exactly that 👍